dashboard/pages/primary_screening/page.py

```diff
@@ -3,9 +3,20 @@
 from dashboard.pages.builders import ProcessPageBuilder
 from dashboard.pages.primary_screening.stages import STAGES
 
+from dashboard.pages.primary_screening.callbacks import register_callbacks
+from dashboard.storage.local import LocalFileStorage
+
+import os
+
 NAME = "Primary Screening"
 register_page(path="/primary-screening", name=NAME, module=__name__)
 
 pb = ProcessPageBuilder(name=NAME)
 pb.add_stages(STAGES)
 layout = pb.build()
+
+fs_dir = os.environ.get("DRUG_SCREENING_DATA_DIR", ".drug-screening-data")
```

Comment:
we will set the fs dir in the `app.py` so that is is changed only once
we shouldnt set this for all pages